### PR TITLE
[15.0][MIG] l10n_ch_base_bank: Migrate to version 15.0

### PR DESCRIPTION
**Pull request by sonhd91**

### CODE REVIEW

**Comment by janikvonrotz** [comment]
@nbessi Would be possible to check this PR?

**Review by ivantodorovich — APPROVED** [review]

**Comment by ivantodorovich** [comment]
Would you consider including this PR?
- https://github.com/OCA/l10n-switzerland/pull/682

ping @santostelmo

**Review by leemannd — APPROVED** [review]

**Comment by sonhd91** [comment]
> Would you consider including this PR?
> 
> * [l10n_ch_base_bank remove isr check #682](https://github.com/OCA/l10n-switzerland/pull/682)
> 
> ping @santostelmo

Hello, I just included PR#682 and remove some related tests case, please check it out

File: l10n_ch_base_bank/README.rst [hunk 1]
```diff
@@ -84,16 +84,23 @@ Authors
...
+    * Vincent Renaville <vincent.renaville@camptocamp.com>
+    * Joël Grand-Guillaume <joel.grandguillaume@camptocamp.com>
+    * Guewen Baconnier <guewen.baconnier@camptocamp.com>
+    * Yannick Vaucher <yannick.vaucher@camptocamp.com>
+    * Akim Juillerat <akim.juillerat@camptocamp.com>
+    * Iryna Vyshnevska <iryna.vyshnevska@camptocamp.com>
+
+* `Trobz <https://trobz.com>`_:
+
+    * Son Ho <sonhd@trobz.com>
```
Comment (yvaucher) [inline l10n_ch_base_bank/README.rst]: Please don't change the structure of contributors, the list was ordered by date of contribution, grouping by company breaks that order.

Comment (sonhd91) [reply l10n_ch_base_bank/README.rst]: Hi, I just updated it, please check again

**Pull request closed without merging**